app/assets/locales/android_translatable_strings.txt

```diff
@@ -329,6 +329,7 @@ title.datum.wrapper=[${0}]
 activity.task.error.generic=Unexpected error! Please try again.
 activity.task.cancelling.title=Cancelling: ${0}
 activity.task.cancelling.message=Cancelling...
+activity.task.cancelled.message=Task Cancelled.

```

Comment:
Would prefer this to be a more user-friendly message, like "Form sending cancelled."